source/_docs/client-ip.md

```diff
@@ -27,6 +27,18 @@ We offer various methods for you to interact with your Pantheon site, so it is i
 
 When using Cloudflare as a stacked CDN or proxy, use the variable `$SERVER["HTTP_CF_CONNECTING_IP"]` instead of `$_SERVER["REMOTE_ADDR"]`.
 
+#### Drupal 7 Domain Access module with Cloudflare
+
+When using Cloudflare in combination with the Domain Access module on Drupal 7, the user's IP address will get cached by the `ip_address()` function incorrectly very early during the bootstrap process. To correct this you can add similar code to your `settings.php` file, above where you include the Domain Access module's `settings.inc` file.
```

Comment:
@daggerhart in the last sentence, are you using "similar" to mean that the example below is similar to the code snippets in the section above, or saying the reader should add something similar to the example below?

Reply:
Great clarifying question. 
I meant, "similar to the code snippet below". I will update the PR.